File: diary_sample/src/main/java/com/example/diary/feature/image/domain/Image.java
package com.example.diary.feature.image.domain;

import com.example.diary.feature.article.domain.Article;
import jakarta.persistence.*;
import lombok.AccessLevel;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Entity
@Getter @NoArgsConstructor(access = AccessLevel.PROTECTED)
public class Image {
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long imageId;
    private String imgName;
    private String imgPath;
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "articleId")
    private Article article;
}

File: diary/src/main/kotlin/sample/diary/domain/image/Image.kt
package sample.diary.domain.image

import jakarta.persistence.*
import sample.diary.domain.article.Article

@Entity
class Image (
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    var imageId:Long,
    @ManyToOne(fetch = FetchType.LAZY)
    var article: Article,
)